### PR TITLE
bug squashing

### PR DESCRIPTION
**Pull request by realcodywburns**
issue #30 fixed column spacing in top navbar and wired in links to footer
issue #32 ch image src to {process.env.PUBLIC_UTL = "imageaddress"
add ToS modal per issue #34

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
         <div class="container">
             <div class="row">
                 <div class="col-md-12">
-                   &copy; 2017 TokenMint.io |<a href="http://www.tokenmint.io/about" target="_blank"> About Us </a> | <i class="fa fa-github" aria-hidden="true"></i> | <i class="fa fa-twitter" aria-hidden="true"></i>
+                   &copy; 2017 TokenMint.io |<a href="https://github.com/ethereumproject/TokenMint/wiki" target="_blank"> About Us </a> | <a href src="https://github.com/ethereumproject/TokenMint"><i class="fa fa-github" aria-hidden="true"></i></a> | <a href src="https://twitter.com/theTokenMint"><i class="fa fa-twitter" aria-hidden="true"></i></a>
```
Comment (elaineo) [inline public/index.html]: we have a twitter acct now? :)

File: src/components/info/ToS.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+            <li> The Software is not a lawyer.</li>
+
+            <li> Your use of the Software does not, in and of itself, create a legally binding contract in any jurisdiction and does not establish a lawyer-client relationship. Your communication with a non-lawyer will not be subject to the attorney-client privilege and (depending on your jurisdiction) may not be entitled to protection as confidential communication.</li>
+
+            <li> The dissemination, distribution, or usage of this software shall not constitute the provision of legal advice within your jurisdiction. Unless you are legally authorized and licensed to do so, you will not use the Software to provide or assist in the provision of legal advice.</li>
+
+            <li> You acknowledge and understand that each jurisdiction has its own particular rules regarding the practice of law. IF YOU USE THIS SOFTWARE TO PROVIDE LEGAL ADVICE YOU MAY BE SUBJECT TO CIVIL AND CRIMINAL LIABILITY. PRACTICING LAW WITHOUT A LICENSE IS A VIOLATION OF CRIMINAL LAW IN SOME JURISDICTIONS. CONSULT A LAWYER LICENSED IN YOUR JURISDICTION IF YOU HAVE ANY QUESTIONS ABOUT WHAT DOES OR DOES NOT CONSTITUTE THE PRACTICE OF LAW.</li>
```
Comment (elaineo) [inline src/components/info/ToS.js]: did you write this ToS yourself? :b

File: public/index.html (hunk 1 above)
Comment (elaineo) [inline public/index.html]: we have a twitter acct now? :)

Comment (realcodywburns) [inline public/index.html]: no we need one. :) @tokenmint already exists so I just added @theTokenmint as a place holder for now. @ChuckSRQ

File: src/components/info/ToS.js (hunk 2 above)
Comment (realcodywburns) [inline src/components/info/ToS.js]: ha ha no as always it is borrowed from elsewhere. It is from the original dao github https://github.com/slockit/DAO

Comment (elaineo) [inline src/components/info/ToS.js]: ah, yes, always a brilliant source of inspiration :)))))

**Pull request merged**